playbooks/group_vars/prometheus/public.yml

```diff
@@ -35,6 +35,8 @@ PROMETHEUS_ALERT_DISK_SPACE_THRESHOLD: 80
 PROMETHEUS_ALERT_HTTP_RESPONSE_TIME_THRESHOLD: 1
 PROMETHEUS_ALERT_HTTP_RESPONSE_FRONTEND_ERROR_THRESHOLD: 70
 
+PROMETHEUS_ALERT_HUEY_QUEUE_SIZE_THRESHOLD: 200
```

Comment:
@shimulch, can you name this variable so that it is obvious that this is related to Ocim's queue?

Reply:
@lgp171188 ``PROMETHEUS_ALERT_OCIM_HUEY_QUEUE_SIZE_THRESHOLD`` seems a better name?

Reply:
@shimulch I think `PROMETHEUS_ALERT_OCIM_HUEY_QUEUE_SIZE_THRESHOLD` should be good. Can you please add change with rename?

Reply:
@0x29a updated.